packages/configuration-recommended/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
@molant proposed to rename this to `configuration-web-recommended`.

(I also think that is a better name)